nvtabular/ops/lambdaop.py

```diff
@@ -65,9 +65,9 @@ def __init__(self, f, dependency=None, label=None):
         self._label = label
 
     @annotate("DFLambda_op", color="darkgreen", domain="nvt_python")
-    def transform(self, columns: ColumnNames, df: DataFrameType) -> DataFrameType:
+    def transform(self, col_selector: ColumnSelector, df: DataFrameType) -> DataFrameType:
```

Comment:
Yeah, it's a reasonable idea. @benfred and I talked about this case, and I think `selector` is somewhat more accurate than `selection`, because `ColumnSelector` represents a potential selection that hasn't yet been concretized. In the previous version of the API, the column names were always a concrete list of actual names, but this PR is shifting toward a somewhat more abstract conception of how columns are selected in order to create room for tag-based selectors (which are inherently a bit more general.)